mdjs-layout/doc/index.md

```diff
@@ -36,20 +29,61 @@ const demoContent = `
     </details>
   </div>
 `;
+
+// context object showing what Backlight generates based on the project meta info
+const demoContext = {};
 ```
 
 ```js preview-story
-export const demo = () => {
-  const configChanged = (ev) => {
-    const change = ev.detail;
-    if (change.key === 'color-scheme-dark-mode') {
-      console.table([change])
-      // Do any changes to sync with your DS dark-mode logic
+import '@divriots/dockit-core/mdjs-layout/load-all';
+import { breakpoints } from '@divriots/dockit-core/mdjs-layout';
+import { html } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { logoSvg } from './logo.svg.js';
+
+const docLayoutTemplate = (content, context) => html`
+  <style>
+    .logo {
+      color: #f59e0b;
     }
-  }
+    .topbar {
+      width: 100%;
+      justify-content: right;
+    }
+    @media screen and (min-width: ${breakpoints.xl}) {

```

Comment:
If the answer is, the first range, then the CSS is correct :)
If it's the second range, it should be +1 pixel 😅

Reply:
I'm not aware of any standard for that. I follow `min-width` strategy, because it's aligned with the mobile-first idea, and therefore all CSS, not only that, is written with the assumption that root selectors are defined for mobile (+ all dimensions if smth suits all) and then the device width grows which I indicate with the help of a certain min-width when rules change.

I think it should be either documented (in my case I should document the usage with `min-width`), or it should be a different token (e.g. `xl: 'min-width: 1030px'`). Given I already included `px` for clarity, I can included min-width too. Wdyt?

Reply:
Yeah I had a small discussion with @thais-divriots  on discord about this and our consensus was the answer is "the first range" so `0-599`, means your CSS is correct. But if we use max-width for some edge-case where it makes sense, we should be aware that it's -1px offset.

I suggested that when writing tokens for breakpoints it makes sense to write min-max queries to prevent people making this error, pseudo:

```js
const tokens = {
  breakpoints: {
    s: 600,
    m: 1024,
    l: 1600,
  },
  queries: {
    s: {
      min: `${this.breakpoints.s.value}px`
      max: `${this.breakpoints.s.value - 1}px`
    }
  }
}
```

```css
@media (min-width: ${queries.s.min}) { }
@media (max-width: ${queries.s.max}) { }
```
or indeed `xl: 'min-width: 1030px'` if you want to go a bit more restrictive, which might make sense. 

Bit off topic this discussion so just to clarify, by no means is this discussion a blocker for the PR :P

Reply:
are these queries compatible with `style-dictionary` and the W3C proposal?

Reply:
Nope probably not, just pseudo code